src/webots/nodes/WbGroup.cpp

```diff
@@ -436,13 +436,19 @@ void WbGroup::readHiddenKinematicParameter(WbField *field) {
 void WbGroup::exportBoundingObjectToX3D(WbVrmlWriter &writer) const {
   assert(writer.isX3d());
 
-  writer << "<Group>";
+  if (isUseNode() && defNode())
+    writer << "<" << x3dName()
+           << " role='boundingObject' USE=\'n" + QString::number(defNode()->uniqueId()) + "\'></" + x3dName() + ">";
```

Comment:
Can we simplify this way?
```suggestion
           << " role='boundingObject' USE=\'n" + QString::number(defNode()->uniqueId()) + "\' />";
```
This would also apply in other classes.

Reply:
I tested it and yes we can.
I looked up at some documentation and found it is a common thing in XML (empty tag) but I found no mention for X3D, but as we are moving away from it, I think we can use empty tags.